src/sql/workbench/browser/modelComponents/divContainer.component.ts

```diff
@@ -169,4 +184,18 @@ export default class DivContainer extends ContainerBase<azdata.DivItemLayout, az
 			this.cancelClick = undefined;
 		}
 	}
+
+	public onMouseOver(): void {
+		// not forcing the mouse in styling on non-clickable divs
+		if (this.clickable) {
+			this.divContainer.nativeElement.style.backgroundColor = this._colorTheme.getColor(colorRegistry.editorHoverBackground).toString();
```

Comment:
I would suggest using a CSS rule with the hover selector instead of manually hooking up the mouseover/mouseout events : 

https://www.w3schools.com/cssref/sel_hover.asp

And then just have a hoverBackground color property that's either the editorHoverBackground color or the default background color